packages/babel-helper-module-transforms/src/rewrite-this.js

```diff
@@ -17,6 +17,11 @@ const rewriteThisVisitor = {
   ClassProperty(path) {
     path.skip();
   },
+  ClassMethod(path) {
+    if (path.node.computed) {
+      //requeue the key child of classProperty here
+    }
```

Comment:
This is a no-op, I don't understand how it fixes the bug :thinking:

Reply:
@nicolo-ribaudo I searched Babel handbook for ClassMethod and found a similar implementation here - https://github.com/jamiebuilds/babel-handbook/blob/master/translations/en/plugin-handbook.md#being-aware-of-nested-structures . Thus, tried to add like mentioned there.

Reply:
Yes, it vists `ClassMethod`, but doesn't do anything when visiting it.
It's similar to how this code visits every array element, but doesn't do anything:
```js
var a = [1, 2, 3];
a.forEach(num => {
  if (num % 2 == 0) {
    // It's even
  }
});
```

I don't think that this issue is fixed, and you can check it by using `@babel/plugin-syntax-class-properties` instead of `@babel/plugin-proposal-class-properties` in the test.

Currently, the test is correctly failing only because `@babel/plugin-proposal-class-properties` moves `this` outside of the computed key.